change the strip to $(STRIP).

From: Xiangfu Liu <xiangfu@sharism.cc>

when cross compile we need use cross comppile strip.

Signed-off-by: Xiangfu Liu <xiangfu@sharism.cc>
---
 Makefile |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 3e1253a..aae3eaa 100644
--- a/Makefile
+++ b/Makefile
@@ -3,6 +3,7 @@
 #
 CC = gcc
 #CC = cc
+STRIP = strip
 
 # Define your optimization flags.  Most compilers understand -O and -O2,
 # Standard (note: Solaris on UltraSparc using gcc 2.8.x might not like this.)
@@ -213,7 +214,7 @@ soundcard.h:
 	fi
 
 install: $(NAME)
-	strip $(BINNAME)$(EXTENSION)
+	$(STRIP) $(BINNAME)$(EXTENSION)
 	install -d $(PREFIX)/bin
 	install -d $(MAN_PREFIX)/man/man6
 	install -c -m 755 $(BINNAME)$(EXTENSION) $(PREFIX)/bin
@@ -226,7 +227,7 @@ uninstall:
 deinstall: uninstall
 
 install_dumb: d$(NAME)
-	strip d$(BINNAME)$(EXTENSION)
+	$(STRIP) d$(BINNAME)$(EXTENSION)
 	install -d $(PREFIX)/bin
 	install -d $(MAN_PREFIX)/man/man6
 	install -c -m 755 d$(BINNAME)$(EXTENSION) $(PREFIX)/bin
@@ -264,7 +265,6 @@ distclean: clean
 	rm -f *core $(SRCDIR)/*core
 	-rm -rf $(distdir)
 	-rm -f $(distdir).tar $(distdir).tar.gz
-	
 realclean: distclean
 
 clobber: distclean
